ts/features/pn/components/F24s.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
```

Comment:
I think a name more along the lines of "F24SectionComponent" would benefit this file; mainly because I feel that "F24s" loses meaning when taken out of context.
what do you think?

Reply:
Done it